packages/parse-cli-args/package.json

```
@@ -35,7 +35,7 @@
     "@pnpm/error": "workspace:2.0.0",
     "@pnpm/find-workspace-dir": "workspace:3.0.1",
     "didyoumean2": "^5.0.0",
-    "nopt": "^5.0.0"
+    "nopt": "npm:@pnpm/nopt@^0.1.0"
```

This should probably use:
```suggestion
    "@pnpm/nopt": "^0.1.0"
```

---

if I do so, I don't think the types from `@types/nopt` will work.